CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+## v1.3.0 (04/03/2020)
+
+**Common**
+
+- Updated dependencies.
+- Improved unit tests coverage.
+- Fixed bugs.
+
+**olp-sdk-fetch**
+
+- Updated `node-fetch` to version 2.6.0.
+
+**olp-sdk-authentication**
+
+- Updated `olp-sdk-fetch` to version 1.3.0.
+
+**olp-sdk-dataservice-api**
+
+- Added functions and models that are generated from the Stream API specification.
+- Added unit tests for all public functions.
+- Added the `isHttpError()` method to the `HttpError` class.
+- Changed `CoverageAPI`. Now, the `dataLevel` parameter is optional.
+
+**olp-sdk-dataservice-read**
+
+- Added the `VersionedLayerClientParams` interface.
+- Deprecated the `VersionedLayerClient` constructor. Instead, use the constructor that creates the `VersionedLayerClient` instance with the help of the `VersionedLayerClientParams` object.
+- Added the `VersionedLayerClient` constructor that creates the `VersionedLayerClient` instance using the `VersionedLayerClientParams` object and lock version.
+- Deprecated the `DataRequest.getVersion()` method.
+- Deprecated the `DataRequest.withVersion()` method.
+- Changed the `VersionedLayerClient.getData` and `VersionedLayerClient.getPartitions` methods. Now, they ignore versions from `DataRequest` and `PartitionsRequest` and use versions from the `VersionedLayerClient` instance. If you initialize `VersionedLayerClient` without any version, on the first call, the latest version is fetched from the service and set in the instance.
+- Added the `SubscribeRequest` class.
+- Changed `StatisticsRequest`: deprecated the `StatisticsRequest.withDatalevel()` method with a string parameter and added the `StatisticsRequest.withDatalevel()` method with a number parameter.
+- Changed `StatisticsRequest.getDataLevel()`. Now, it returns one of the following values: string, number, or undefined.
+- Added `StreamLayerClient`.
+- Added the `StreamLayerClient.subscribe()` method.
+- Added the `StreamLayerClient.poll()` method.
+- Added the `StreamLayerClient.seek()` method.
+- Added the `StreamLayerClient.getData()` method.
+- Added the `StreamLayerClient.unsubscribe()` method.
```

Comment:
Merge and describe what is the StreamLayerClient to be used for, e.g. to consume messages from a stream layer in serial or parallel mode, etc.

Reply:
updated